sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioFS.scala

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.nvidia.spark.rapids
+
+import scala.collection.JavaConverters.mapAsScalaMapConverter
+import scala.collection.mutable
+
+import alluxio.AlluxioURI
+import alluxio.conf.{AlluxioProperties, InstancedConfiguration, PropertyKey}
+import alluxio.grpc.MountPOptions
+
+/**
+ * interfaces for Alluxio file system.
+ * Currently contains interfaces:
+ *   get mount points
+ *   mount
+ */
+class AlluxioFS extends Arm {
+  private var masterHost: Option[String] = None
+  private var masterPort: Option[Int] = None
+  private var alluxioUser: String = ""
+  private var s3AccessKey: Option[String] = None
+  private var s3SecretKey: Option[String] = None
+
+  def setHostAndPort(masterHost: Option[String], masterPort: Option[Int]): AlluxioFS = {
+    this.masterHost = masterHost
+    this.masterPort = masterPort
+    this
```

Comment:
why do we need to return this here?  You have an instance so just set it

Reply:
Done.